### PR TITLE
Fix the paint brush

### LINKED ISSUE #30: Doodle pencil is confusing. Use a paint brush instead!
Change the icon

### PR DESCRIPTION
**Pull request by robinmaypanpan**
Fixes #30

### CODE REVIEW

File: app/javascript/mastodon/features/ui/components/doodle_modal.js [hunk 1]
```diff
@@ -542,72 +542,72 @@ export default class DoodleModal extends ImmutablePureComponent {
...
+              </div>
             </div>
-            <div>
-              <select aria-label='Canvas size' onInput={this.changeSize} defaultValue={this.size}>
-                { Object.values(mapValues(DOODLE_SIZES, (val, k) =>
-                  <option key={k} value={k}>{val[2]}</option>
-                )) }
-              </select>
+            <div className='doodle-toolbar'>
+              <IconButton icon='pencil' title='Draw' label='Draw' onClick={this.setModeDraw} size={18} active={this.mode === 'draw'} inverted />
```
Comment (aarchiba) [inline app/javascript/mastodon/features/ui/components/doodle_modal.js:584]: should this be paintbrush?

**Review by aarchiba — APPROVED** [review]
I can't really tell what's going on with the div rearrangements (just moving buttons to the top?) but everything else looks okay.

**Comment by robinmaypanpan** [comment]
Closing this for now.  I have more stuff I wanna add to 1.5.1...

File: app/javascript/mastodon/features/ui/components/doodle_modal.js (hunk 1 above)
Comment (robinmaypanpan) [reply app/javascript/mastodon/features/ui/components/doodle_modal.js:584]: Nope!

**Pull request closed without merging**